models/stg_reddit_ads__account_report.sql

```diff
@@ -15,7 +15,6 @@ fields as (
                 staging_columns=get_account_report_columns()
             )
         }}
-        
```

Comment:
Small whitespace removal suggestion.

Please apply the same changes to the other staging models.